lib/Command/ConfigReport.php

```diff
@@ -30,8 +30,25 @@
  */
 class ConfigReport extends Command {
 
-	public function __construct() {
+	/**
+	 * @var ReportDataCollector
+	 */
+	private $reportDataCollector;
 
+	public function __construct() {
```

Comment:
> Why would it generate a full report on instantiation? o.O

Well, not a full report but if it instantiates ReportDataCollector (and fetches all users wíth `getUsers()` !) for any command that is run even unrelated, it's a waste of time and memory. 

A way to test it would be to set a breakpoint in the constructor and run an unrelated command like "occ setting:user" and see if it goes there. If it does, then we need to make this here more lazy.